core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/RepairStateImpl.java

```diff
@@ -206,8 +205,8 @@ private long partiallyRepairedTableRepairedAt(final long maxRepairedAt, final Re
     private long newTableRepairedAt()
     {
         long runIntervalInMs = myRepairConfiguration.getRepairIntervalInMs();
-        long minimumRepairWait = Math.min(runIntervalInMs, TimeUnit.DAYS.toMillis(1));
-        long assumedRepairedAt = System.currentTimeMillis() - runIntervalInMs + minimumRepairWait;
+        long newTableSlideOff = myRepairConfiguration.getNewTableSlideOff();
```

Comment:
Refactor variable name and method name! No slideOff etc. Why not initialDelayInMs, as for instance "getRepairIntervalInMs"?